src/components/reusable/sideDrawer/sideDrawer.scss

```diff
@@ -86,9 +74,10 @@ form {
 
 .body {
   margin: 0 32px;
-  padding-right: 24px;
+  padding: 32px;
```

Comment:
The body text is no longer aligned with the title.

Reply:
✅